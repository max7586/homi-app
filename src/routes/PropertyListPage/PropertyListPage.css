.PropertyListPage {
  border-radius: 50px 0px 50px 0px;
  /* display: flex; */
  /* justify-content: space-evenly; */
  /* flex-wrap: wrap; */
}
