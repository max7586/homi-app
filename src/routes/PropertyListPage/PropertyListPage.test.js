import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import PropertyListPage from './PropertyListPage';
import { BrowserRouter } from 'react-router-dom'


describe('PropertyListPage component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <BrowserRouter>
        <PropertyListPage />
      </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer.create(<BrowserRouter>
        <PropertyListPage />
      </BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();  
  });


});