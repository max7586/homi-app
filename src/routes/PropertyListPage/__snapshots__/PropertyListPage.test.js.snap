// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyListPage component renders the UI as expected 1`] = `
<section
  className="Section Section--list PropertyListPage"
/>
`;
